packages/framework/client-logger/external-telemetry/api-report/external-telemetry.api.md

```diff
@@ -0,0 +1,69 @@
+## API Report File for "@fluidframework/external-telemetry"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ICriticalContainerError } from '@fluidframework/container-definitions';
+import type { IFluidContainer } from '@fluidframework/fluid-static';
+
+// @beta
+export interface ContainerConnectedTelemetry extends IContainerTelemetry {
+    eventName: "fluidframework.container.connected";
+}
+
+// @beta
+export interface ContainerDisconnectedTelemetry extends IContainerTelemetry {
+    eventName: "fluidframework.container.disconnected";
+}
+
+// @beta
+export interface ContainerDisposedTelemetry extends IContainerTelemetry {
+    error?: ICriticalContainerError;
+    eventName: "fluidframework.container.disposed";
+}
+
+// @beta
+export type ContainerTelemetryEventName = (typeof ContainerTelemetryEventNames)[keyof typeof ContainerTelemetryEventNames];
+
+// @beta
+export const ContainerTelemetryEventNames: {
+    readonly CONNECTED: "fluidframework.container.connected";
+    readonly DISCONNECTED: "fluidframework.container.disconnected";
+    readonly DISPOSED: "fluidframework.container.disposed";
+    readonly HEARTBEAT: "fluidframework.container.heartbeat";
+};
+
+// @beta
+export type ExternalTelemetryEventName = ContainerTelemetryEventName;
+
+// @beta
+export interface IContainerTelemetry extends IExternalTelemetry {
+    containerId?: string;
+    containerInstanceId: string;
+    eventName: ContainerTelemetryEventName;
+}
+
+// @beta
+export interface IExternalTelemetry {
+    eventName: ExternalTelemetryEventName;
+}
+
+// @beta
+export interface ITelemetryConsumer {
+    consume(event: IExternalTelemetry): any;
+}
+
+// @beta
+export const startTelemetry: (config: TelemetryConfig) => void;
+
+// @beta
+export interface TelemetryConfig {
+    consumers: ITelemetryConsumer[];
+    container: IFluidContainer;
+    containerId: string;
+}
+
+// (No @packageDocumentation comment for this package)
```

Comment:
Let's add a brief `@packageDocumentation` comment in the root `index.ts` file. That's what will end up showing on the landing page for the package in the generated API docs. This comment will go away if you add it.